lib/features/sks-menu/presentation/widgets/sks_menu_loading.dart

```diff
@@ -0,0 +1,70 @@
+import "package:flutter/material.dart";
+import "package:lottie/lottie.dart";
+
+import "../../../../../theme/app_theme.dart";
+import "../../../../../utils/context_extensions.dart";
+import "../../../../gen/assets.gen.dart";
+
+class LottieLoading extends StatefulWidget {
```

Comment:
I'd think about better naming, that reflects where such lottie animation is used, eg. "SKSMenuLottieAnimation" or sth like that.